src/display/lv_display.c

```diff
@@ -1031,6 +1031,37 @@ void lv_display_rotate_area(lv_display_t * disp, lv_area_t * area)
     }
 }
 
+uint32_t lv_display_get_draw_buf_size(lv_display_t * disp)
+{
+    if(!disp) disp = lv_display_get_default();
+    if(!disp) return 0;
```

Comment:
Question: I noticed for invalid cases, for example invalid display pointer we return 0 sized buffer, would not be better to return an invalid size token instead, or if by doing this the API would break an LOG_ERROR on such cases are very welcome to track why invalid display was passed to this function.

Reply:
Its really a question of general API design. There are two ways of doing it:
`int foo(lv_display_t*,  uint32_t*)`
and `uint32_t foo(lv_display_t*)` and returning a special value.

I have not seen much use of the first one inside the codebase so I went with the second approach :^)

Reply:
Make sense so far, I think error handling would be a separate topic to dicuss, so I will resolve that :)